public/css/module.less

```diff
@@ -1518,15 +1531,15 @@ input[type=submit].icon-button {
 }
 
 .breadcrumb li a:before {
-  border-left: 1.2em solid white;
-  border-left: 1.2em solid @icinga-blue;
+  border-left: 1.2em solid;
+  .var(border-left-color, icinga-blue);
   margin-left: 1px;
   z-index: 1;
 }
 
 .breadcrumb li a:after {
-  border-left: 1.2em solid @icinga-blue;
-  border-left: 1.2em solid white;
+  border-left: 1.2em solid;
+  .var(border-left-color, "white");
```

Comment:
Please use `body-bg-color` here. Otherwise it looks glitchy in light mode.

![Screen Shot 2021-11-23 at 17 04 36](https://user-images.githubusercontent.com/6977434/143059902-e64bb662-d2ca-4b45-abdb-c08bf76a7783.png)